README.md

```diff
@@ -1,2 +1,59 @@
-# outreach
-A Node.js community growth and support initiative
+<p align="center">
+  <br>
+  <a href="https://nodejs.dev">
+    <img src=".github/images/nodejslogo.png" width="200"/>
+  </a>
+</p>
+
+<h1 align="center"><a href="https://node.js.dev"> Outreach Initiative</a></h1>
+
+<p align="center">The Outreach Initiative is aimed to provide help, guidance, and spotlight to the Node.js community.</p>
+
+<p align="center">
+  <a title="MIT License" href="LICENSE">
+    <img src="https://img.shields.io/github/license/gridsome/gridsome.svg?style=flat-square&label=License&colorB=6cc24a">
+  </a>
+  <a title="Follow on Twitter" href="https://twitter.com/Node.js">
+    <img src="https://img.shields.io/twitter/follow/Node.js.svg?style=social&label=Follow%20@Node.js">
+  </a>
+  <br>
+  <br>
+</p>
+
+## 🌳 Node.js Outreach Initiative
+
+The **Outreach Initiative** is a collaborative effort to help people in the Node.js community by introducing consistent workflows around avenues of community engagements like events, content, and open source contributions.
+
+Node.js is emerging as a universal development framework and has grown to a user base of more than **10 million developers**. At the same time, Node users continue to report positive business impact, including improving developer productivity, satisfaction, and a lower cost. Moreover, considering the aggressive competition, the outreach team strives to publicize the work of the Node.js developers in the tech world.
+
+## 🎯 Initiative Goals
+
+The Node.js outreach initiative is in its early stages. We are currently focused on improving community outreach workflows around the following three goals:
+
+- [Community Events](#Community-Events)
+- [Community Content](#Community-Content)
+- [Community Contributions](#Community-Contributions)
+
+### Community Events
+
+Community events goal includes Node.js related meetups, conferences, workshops, and more. We are currently figuring out ways to promote Node speakers, events, and build consistent meetups workflow to increase participation in such events.
+
+> [Discuss the work-in-progress ideas for community events here →](https://github.com/nodejs/outreach/issues/7)
+
+### Community Content
+
+Community content goal includes everything related to writing, curating, and sharing content around Node.js.
+
+> [Discuss the work-in-progress ideas for community content here →](https://github.com/nodejs/outreach/issues/8)
+
+### Community Contributions
+
+Community contributions goal includes building resources for improving the workflow for existing as well as new open source contributors for the Node.js Foundation. Consistent contribution workflow will help attract more community contributions.
```

Comment:
+1 to this goal.

The Node.js Foundation and the JSF have merged to form the OpenJS Foundation. The Foundation provides support to multiple projects, including Node.js. I'd suggest specifying the Node.js project since the outreach is meant for this community.

```suggestion
Community contributions goal includes building resources for improving the workflow for both existing and new open source contributors to the Node.js project. Consistent contribution workflow will help attract more community contributions.
```

Reply:
Makes sense. 💯